docker/cloud.Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM pangeo/pangeo-notebook:latest
+
+RUN pip install pan3d[all] imageio trame-jupyter-extension
+RUN pip uninstall -y vtk
+RUN pip install --extra-index-url https://wheels.vtk.org vtk-osmesa
+
+COPY ./examples /shared/pan3d_examples
```

Comment:
Just a sanity check: Is `/shared` the location that Jim Colliander told us to put our shared files? Seems right, but I don't have anything in my notes from our last meeting with him.

Reply:
I don't remember Jim specifying anything about the file locations, but Yuvi had stated in his last email that "if you have something in the docker *image*, my suggestion is to put that somewhere other than `$HOME` (like `/srv/shared`), and that would automatically be accessible to all users. This is because just $HOME is overriden by persistent storage, but everything else works as normal." 

Maybe we could try `/srv/shared` as he suggests, but by this description `/shared` should work. However, the behavior we observed is that this location is overwritten in the way he says the `$HOME` location is. Not sure what location is best to make the examples available to all users by default (and showing up in their file tree). In my recollection, we haven't gotten a straight answer about that.